<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Pos 1 : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="pos1SpinBox">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setting1Btn">
          <property name="text">
           <string>Setting1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Pos 2 : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="pos2SpinBox">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setting2Btn">
          <property name="text">
           <string>Setting2</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Pos 3 : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="pos3SpinBox">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setting3Btn">
          <property name="text">
           <string>Setting3</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Pos 4 : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="pos4SpinBox">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setting4Btn">
          <property name="text">
           <string>Setting4</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Cutoff : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="cutoffSpinBox"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Threshold : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="threshSpinBox"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="motionACb">
          <property name="text">
           <string>Motion A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="motionBCb">
          <property name="text">
           <string>Motion B</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="collisionCb">
          <property name="text">
           <string>CollisionDetect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="readyBtn">
          <property name="text">
           <string>READY</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="runBtn">
          <property name="text">
           <string>RUN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="quitBtn">
          <property name="text">
           <string>QUIT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="emergen">
          <property name="text">
           <string>EMERGENCY</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>819</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
